samples/config_helper_vulkan.cc

```diff
@@ -920,6 +920,15 @@ amber::Result ConfigHelperVulkan::CheckVulkanPhysicalDeviceRequirements(
     vkGetPhysicalDeviceFeatures2KHR(physical_device, &features2);
     available_features_ = features2.features;
 
+    // Just having the feature does not necessarily mean that the feature is available. We have to
```

Comment:
Recommend changing "feature" to "extension" like this:

> Just having the extension does not....

Reply:
done.